packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -19,11 +19,16 @@ class LinuxDoctorValidator extends DoctorValidator {
     ValidationType validationType = ValidationType.installed;
     final List<ValidationMessage> messages = <ValidationMessage>[];
     /// Check for a minimum version of Clang.
-    final ProcessResult clangResult = await processManager.run(const <String>[
-      'clang++',
-      '--version',
-    ]);
-    if (clangResult.exitCode != 0) {
+    ProcessResult clangResult;
+    try {
+      clangResult = await processManager.run(const <String>[
```

Comment:
This does do a bit more than `processManager.canRun` - but is that intended?  Is `canRun` not good enough here?

Reply:
I could be wrong but I've seen `canRun` throw argument errors too